Tools/patchcheck/patchcheck.py

```diff
@@ -23,7 +23,7 @@
 
 def n_files_str(count):
     """Return 'N file(s)' with the proper plurality on 'file'."""
-    return "{} file{}".format(count, "s" if count != 1 else "")
+    return f"{count} file{'s' if count != 1 else ''}"
```

Comment:
I quite like the 's' variable approach, it's more readable:

```suggestion
    s = "s" if count != 1 else ""
    return f"{count} file{s}"
```